src/framapp-polymer-app/fp-data-store.html

```diff
@@ -0,0 +1,34 @@
+<link rel="import" href="../../bower_components/iron-ajax/iron-ajax.html">
+
+<dom-module id="fp-data-store">
+    <template>
+        <iron-ajax auto
+            url=[[sourceUrl]]
+            handle-as="json"
+            on-response="_handleResponse"></iron-ajax>
+    </template>
+    <script>
+        Polymer({
+
+            is: 'fp-data-store',
+            properties: {
+                categories: {
+                    type: Array,
+                    notify: true,
+                },
```

Comment:
this is fine but I'd go with [computed propery](https://www.polymer-project.org/1.0/docs/devguide/observers#define-a-computed-property)